include/internal/catch_common.hpp

```diff
@@ -22,7 +24,7 @@ namespace Catch {
         return s.find( infix ) != std::string::npos;
     }
     void toLowerInPlace( std::string& s ) {
-        std::transform( s.begin(), s.end(), s.begin(), ::tolower );
+        std::transform( s.begin(), s.end(), s.begin(), static_cast<int(*)(int)>(std::tolower) );
```

Comment:
Can't use lambdas due to C++-03 compatibility.
I think the answer might be along similar lines, though - go via a(n out of line) forwarding function.